base/src/com/google/idea/blaze/base/qsync/BazelInfoHandler.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2024 The Bazel Authors. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.idea.blaze.base.qsync;
+
+import com.google.idea.blaze.base.bazel.BuildSystem;
+import com.google.idea.blaze.base.command.info.BlazeInfo;
+import com.google.idea.blaze.base.sync.SyncScope;
+import com.google.idea.blaze.exception.BuildException;
+
+public class BazelInfoHandler {
+    private final BuildSystem.BuildInvoker buildInvoker;
+
+    BazelInfoHandler(BuildSystem.BuildInvoker buildInvoker) {
+        this.buildInvoker = buildInvoker;
+    }
+
+    BlazeInfo getBazelInfo() throws BuildException {
+        try {
+            return  buildInvoker.getBlazeInfo();
```

Comment:
Formatting is wrong here too, as far as I know we are following [this](https://github.com/google/styleguide/blob/gh-pages/intellij-java-google-style.xml) code style.